pkgs/development/python-modules/klayout/default.nix

```diff
@@ -0,0 +1,41 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  curl,
+  cython,
+  expat,
+  libpng,
+  setuptools,
+}:
+
+buildPythonPackage rec {
+  pname = "klayout";
+  version = "0.29.8";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-21EPhFb/JMZdyuHDXIxhnLTpHUPxKU24cnodH9oX2q8=";
+  };
+
+  build-system = [
+    cython
+    setuptools
+  ];
+
+  buildInputs = [
+    curl
+    expat
+    libpng
+  ];
+
+  pythonImportsCheck = [ "klayout" ];
+
```

Comment:
Why no tests for this package?

Reply:
`klayout` is a full C++ application with python and ruby interfaces. There is a set of tests for the full application. However, this library is a subset of the full package only containing the python interface, and I believe that it comes with no separate tests.